### PR TITLE
doc review

### PR DESCRIPTION
**Pull request by hmeAtos**

### CODE REVIEW

File: resources/documentation/learn_ansible.rst [hunk 1]
```diff
@@ -5,26 +5,25 @@ Learn Ansible
...
 Ansible.
 
-This section tries to provide key knowledge to at least be able to manipulate
+This section provides key knowledge to, at least, be able to manipulate
 and edit the stack to your needs.
 
-It is assumed here that your system is already installed, and that Ansible has
-also been installed. It is also assumed that **BlueBanquise** stack has not been
-installed yet, and so Ansible is freshly installed without modifications.
+It is assumed that your system and Ansible are installed. It is also assumed that the **BlueBanquise** stack is NOT yet
```
Comment (btravouillon) [inline resources/documentation/learn_ansible.rst:11]: Can you keep the 80 columns layout? It helps to read in the terminal.

Comment (hmeAtos) [reply resources/documentation/learn_ansible.rst:11]: Of course @btravouillon

**Comment by oxedions** [comment]
Dear @hmeAtos 

Do you think I can start reviewing or do you plan other changes on the files already corrected ?

**Comment by hmeAtos** [comment]
> Dear @hmeAtos
> 
> Do you think I can start reviewing or do you plan other changes on the files already corrected ?

yes you can @oxedions 
regards

File: resources/documentation/configure_bluebanquise.rst [hunk 2]
```diff
@@ -161,10 +161,9 @@ Other nodes
 ^^^^^^^^^^^
 
 Now, review compute and login nodes in their respective
-cluster/nodes/computes.yml and cluster/nodes/logins.yml files. Same rules
-apply. You can also add more nodes, or if you have for example multiple type
-of equipment for computes nodes or login nodes, add another equipment group
-this way:
+**cluster/nodes/computes.yml** and **cluster/nodes/logins.yml**. The same rules
+apply. You can add more nodes. You can add equipment groups for exemple if you have multiple types
```
Comment (bouriquet) [inline resources/documentation/configure_bluebanquise.rst:165]:
```suggestion
apply. You can add more nodes. You can add equipment groups for example if you have multiple types
```

File: resources/documentation/customize_inventory.rst [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+
+Changing the Time Zone
+----------------------
+
+To identify which time zone is closest to your present location, use the ``timedatectl`` command with the ``list-timezones`` command line option.
+For example, to list all available time zones in Europe, type:
+
+.. code-block:: shell
+
+ timedatectl list-timezones | grep Paris
```
Comment (bouriquet) [inline resources/documentation/customize_inventory.rst:21]:
```suggestion
 timedatectl list-timezones | grep -i europe
```

File: resources/documentation/customize_inventory.rst [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+
+Disabling the SSH strict host key checking
+------------------------------------------
+
+.. warning::
+   In BlueBanquise it is possible to disable the strict host key checking in the inventory.
+   **By default it is an activated option.**
+
+As a reminder, if this flag is set to 'yes', ssh will never automatically add host keys to the ``~/.ssh/known_hosts`` file, and refuses to connect to hosts whose host key has changed.
+This provides maximum protection against trojan horse attacks.
```
Comment (bouriquet) [inline resources/documentation/customize_inventory.rst:85]:
```suggestion
This provides protection against MIM attacks.
```

**Comment by oxedions** [comment]
@johnnykeats Could you have a look on this ?

**Comment by johnnykeats** [comment]
@hmeAtos 
Ok I commit on your PR ?

**Comment by hmeAtos** [comment]
> Ok I commit on your PR ?

Yes you can.

**Pull request closed without merging**